### ISSUE TITLE
Crash when remote server uses IPv6

### ISSUE THREAD

**Issue by wooky** [wooky/ottd-prayer#2]
```
Traceback (most recent call last):
  File "main.py", line 4, in <module>
  File "ottd_prayer/main.py", line 48, in main
  File "asyncio/runners.py", line 44, in run
  File "asyncio/base_events.py", line 646, in run_until_complete
  File "ottd_prayer/main.py", line 40, in main_async
  File "ottd_prayer/server_connector.py", line 22, in connect_to_server
  File "ottd_prayer/client_runner.py", line 19, in run_client
  File "asyncio/base_events.py", line 1025, in create_connection
  File "asyncio/base_events.py", line 1404, in _ensure_resolved
  File "asyncio/base_events.py", line 860, in getaddrinfo
  File "concurrent/futures/thread.py", line 58, in run
  File "socket.py", line 955, in getaddrinfo
socket.gaierror: [Errno -2] Name does not resolve
[28860] Failed to execute script 'main' due to unhandled exception!
```
The coordinator returns an IPv6 address enclosed in square brackets (e.g. `[::1]`), so passing that to loop.create_connection chokes it. These brackets needs to be stripped out ahead of time. To be extra safe, check the OpenTTD source code to find out what it does.

**Comment by wooky** [comment]
This issue seems to occur only Linux OSs. C's getaddrinfo does not like square brackets. It doesn't look like the OpenTTD strips those those brackets out (or, at least I can't find it), but it should be done for the bot.

**Comment by wooky** [comment]
Fixed as of 2f35dcbe231fcb943c00ccdaaebcb87e6218b7da
